apps/worker/src/app/workflow/usecases/send-message/send-message-push.usecase.ts

```diff
@@ -113,81 +120,103 @@ export class SendMessagePush extends SendMessageBase {
         Object.values(PushProviderIdEnum).includes(chan.providerId as PushProviderIdEnum)
       ) || [];
 
-    const messagePayload = Object.assign({}, command.payload);
-    delete messagePayload.attachments;
-
     if (!pushChannels.length) {
-      await this.sendErrors(command);
```

Comment:
Moved above to try to fail as soon as possible and because the calculations for pushChannels are now together.